File: csharp/byps/BRemoteRegistry.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace byps
{
    public interface BRemoteRegistry
    {
        BRemote getRemote(BTargetId serverId, BTargetId clientId, int remoteId);
    }
}

File: java/byps/src/byps/BRemoteRegistry.java
package byps;

/* USE THIS FILE ACCORDING TO THE COPYRIGHT RULES IN LICENSE.TXT WHICH IS PART OF THE SOURCE CODE PACKAGE */

import byps.BException;
import byps.BRemote;
import byps.BTargetId;

public interface BRemoteRegistry {
	public BRemote getRemote(BTargetId clientId, int remoteId) throws BException;
}
